docs/user/dql/functions.rst

```diff
@@ -1231,6 +1231,7 @@ Description
 >>>>>>>>>>>
 
 Usage: date_format(date, format) formats the date argument using the specifiers in the format argument.
+If an argument of type TIME is provided, the local date is used.
```

Comment:
Update `date_format` PPL doc too

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/204/commits/c44602293107b2037d35f9c09138451a73c3bff1